smf/clickhouse-admin-single/manifest.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='clickhouse-admin-single'>
+
+  <service name='oxide/clickhouse-admin-single' type='service' version='1'>
+  <create_default_instance enabled='true' />
+
+  <dependency name='multi_user' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/milestone/multi-user:default' />
+  </dependency>
+
+  <dependency name='zone_network_setup' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/oxide/zone-network-setup:default' />
+  </dependency>
+
+  <exec_method type='method' name='start'
+    exec='ctrun -l child -o noorphan,regent /opt/oxide/omicron-clickhouse-admin/bin/clickhouse-admin-single run --config /var/svc/manifest/site/clickhouse-admin-single/config.toml --http-address %{config/http_address} --listen-address %{config/ch_address} --binary-path %{config/ch_binary}  &amp;'
```

Comment:
Same here. I think we can remove `--config /var/svc/manifest/site/clickhouse-admin-single/config.toml`